### PR TITLE
ECSOPS-213: Update plugins to latest

### PR DESCRIPTION
**Pull request by ianbrown9475**

### CODE REVIEW

File: download_plugins.py [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-import yaml
 import sys
 from logging import debug
 from textwrap import dedent
-from distutils.version import LooseVersion
+
+import requests
+import yaml
+
+import utils
```
Comment (tsoliangwu0130) [inline download_plugins.py]: How did you order the import commands now? Is there any reason to order like this?

File: download_plugins.py [hunk 2]
```diff
@@ -23,46 +24,22 @@ def get_dependencies(plugin):
...
-                        url=plugin_base_url, plugin=plugin,
-                        version=version
-                    ))
+# Download latest version of plugin from jenkins update server.
+def download_plugin(plugin):
+    if plugin in stored_plugins.keys():
+        return
+    download_url = "{url}/latest/{plugin}.hpi".format(
+                        url=plugin_base_url, plugin=plugin
+                    )
```
Comment (tsoliangwu0130) [inline download_plugins.py]: I suggest using [f-string](https://docs.python.org/3/reference/lexical_analysis.html#formatted-string-literals) (Literal String Interpolation) to format string in Python 3.6 or above. It should be able to make the code looks better :)

**Review by tsoliangwu0130 — CHANGES_REQUESTED** [review]

File: download_plugins.py (hunk 1 above)
Comment (ianbrown9475) [reply download_plugins.py]: They were ordered using "optimize imports" in PyCharm. Pretty sure it follows pep 8 https://www.python.org/dev/peps/pep-0008/#imports specifically:
> Imports should be grouped in the following order:
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.
>
> You should put a blank line between each group of imports.

File: download_plugins.py (hunk 2 above)
Comment (ianbrown9475) [reply download_plugins.py]: Oh neat. I was wondering if there was an easy way in python to just insert variables in a string and it looks like you can do that as well as a lot of other stuff with this method, nice.

File: download_plugins.py (hunk 1 above)
Comment (tsoliangwu0130) [reply download_plugins.py]: Good to know, thanks.

**Review by tsoliangwu0130 — APPROVED** [review]

**Pull request merged**